src/v2/view-builder/views/password/EnrollAuthenticatorPasswordView.js

```diff
@@ -1,10 +1,10 @@
-import { loc, View } from 'okta';
+import { loc } from 'okta';
```

Comment:
All though it's not part of the PR but we should not use a function that is part of v1 widget.. `getPasswordComplexityDescriptionForHtmlList` 

Either bring it in v2 or write a new one.. You can create a jira for this as tech debt and assign to your team

Reply:
Created ticket to track this: https://oktainc.atlassian.net/browse/OKTA-484804